simulationdataschema/model_system_review.py

```diff
@@ -0,0 +1,864 @@
+        """,
+    )
+
+    hall_symbol = Quantity(
+        type=str,
+        description="""
+        Hall symbol for this system describing the minimum number of symmetry operations,
+        in the form of Seitz matrices, needed to uniquely define a space group. See
+        https://cci.lbl.gov/sginfo/hall_symbols.html. Examples:
+            - `F -4 2 3`,
+            - `-P 4 2`,
+            - `-F 4 2 3`.
+        """,
+    )
+
+    point_group_symbol = Quantity(
+        type=str,
+        description="""
+        Symbol of the crystallographic point group in the Hermann-Mauguin notation. See
+        https://en.wikipedia.org/wiki/Crystallographic_point_group. Examples:
+            - `-43m`,
+            - `4/mmm`,
+            - `m-3m`.
+        """,
+    )
+
+    space_group_number = Quantity(
+        type=np.int32,
+        description="""
+        Specifies the International Union of Crystallography (IUC) space group number of the 3D
+        space group of this system. See https://en.wikipedia.org/wiki/List_of_space_groups.
+        Examples:
+            - `216`,
+            - `123`,
+            - `225`.
+        """,
+    )
+
+    space_group_symbol = Quantity(
+        type=str,
+        description="""
+        Specifies the International Union of Crystallography (IUC) space group symbol of the 3D
+        space group of this system. See https://en.wikipedia.org/wiki/List_of_space_groups.
+        Examples:
+            - `F-43m`,
+            - `P4/mmm`,
+            - `Fm-3m`.
+        """,
+    )
+
+    strukturbericht_designation = Quantity(
+        type=str,
+        description="""
+        Classification of the material according to the historically grown and similar crystal
+        structures ('strukturbericht'). Useful when using altogether with `space_group_symbol`.
+        Examples:
+            - `C1B`, `B3`, `C15b`,
+            - `L10`, `L60`,
+            - `L21`.
+
+        Extracted from the AFLOW encyclopedia of crystallographic prototypes.
+        """,
+    )
+
+    prototype_formula = Quantity(
+        type=str,
+        description="""
+        The formula of the prototypical material for this structure as extracted from the
+        AFLOW encyclopedia of crystallographic prototypes. It is a string with the chemical
+        symbols:
+            - https://aflowlib.org/prototype-encyclopedia/chemical_symbols.html
+        """,
+    )
+
+    prototype_aflow_id = Quantity(
+        type=str,
+        description="""
+        The identifier of this structure in the AFLOW encyclopedia of crystallographic prototypes:
+            http://www.aflowlib.org/prototype-encyclopedia/index.html
+        """,
+    )
+
+    atomic_cell_ref = Quantity(
+        type=AtomicCell,
+        description="""
+        Reference to the AtomicCell section that the symmetry refers to.
+        """,
+        a_eln=ELNAnnotation(component="ReferenceEditQuantity"),
+    )
```

Comment:
Is this still necessary in our new design?

Reply:
I think we should leave more things as ELN annotations, but I didn't tackle it fully yet.

Reply:
I'm completely fine with it being an ELN section. I more so meant the reference part...
This ties in with the multiple `AtomicCell`s. I think we can have a single one that explains its full `Symmetry`, but that's what I'd like to discuss with Lauri.

Just leave a note about the reference.